usvm-core/src/main/kotlin/org/usvm/StateForker.kt

```diff
@@ -144,7 +145,9 @@ object WithSolverStateForker : StateForker {
         newConstraintToForkedState: UBoolExpr,
         stateToCheck: StateToCheck,
     ): T? {
-        val constraintsToCheck = state.pathConstraints.clone()
+        val thisOwnership = MutabilityOwnership()
+        val cloneOwnership = MutabilityOwnership()
+        val constraintsToCheck = state.pathConstraints.clone(thisOwnership, cloneOwnership)
```

Comment:
I don't like current ownership flow:
1. We firstly change the ownership of `state.pc` without modification of `state` ownership
2. We use ownership from pc during the state clone operation. So, we use `pc` as a source of ownership

Reply:
Anyway we have to create some ownership `$o_1$` here for next `constraints += condition` operation. We can manually reset `pc.ownership` in `state.clone` with new ownership `o_2`, but somewhere in cloned constraints will be contained record with `$o_1$`. Is it ok? The only problem i see that temporary ownership `$o_1$` can confuse during debug.